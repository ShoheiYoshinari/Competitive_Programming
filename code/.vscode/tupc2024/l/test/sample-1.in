3
8 3
20 24
998236771 998244353
