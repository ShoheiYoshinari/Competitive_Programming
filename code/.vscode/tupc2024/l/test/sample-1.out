2 1 3
3 5 76 24
1 998244353
